setup.py

```diff
@@ -20,56 +20,187 @@
     sys.exit(2)
 
 requirements = [
-    "Django>=3.2,<3.3",  # when upgrading, upgrade also django-two-factor-auth!
-    "pytz>=2013b,<=2021.1",
-    "sqlalchemy<1.5",
-    "beautifulsoup4<4.10",
-    "PyYAML<5.5",
-    "python-dateutil<2.9",
-    "django-two-factor-auth==1.13.2",
-    "django-formtools>=2.2,<=2.3",
-    "django-registration-redux>=2.6,<=2.9",
-    "Celery==4.4.7",
-    "coreapi>=2.3.0,<2.4",
-    "dj-pagination==2.5",
-    "django-compressor==2.4.1",  # latest version
-    "pygments<2.6",
-    "django-libsass>=0.7,<=0.8",
-    "django-debug-toolbar>=3.0,<=3.3",
-    "django-extensions>=3.0,<=3.2",  # Django 2.2
-    "djangorestframework>=3.10,<3.13",
-    "werkzeug<1.1",
-    'pytest==4.6.11',
-    'pytest-metadata==1.11.0',
-    'pytest-django==3.10.0',
-    'pytest-html==1.22.1',
-    'pytest-xdist==1.34.0',
-    'pytest-cov>=2.11,<2.12',
-    'requests<3',
-    "fpdf<1.8",
-    "unicodecsv<0.15",
-    "shortuuid<1",
-    "dnslib<0.10",
-    "bleach>=3.1.0,<3.2",
-    "chardet<4.1",
-    "django-gravatar2<1.5",
-    "django-mptt>=0.10,<=0.12",
-    "mistune<0.9",
-    "pika<1.3",
-    "raven<6.11",
-    "unidecode<1.3",
-    "sentry-sdk",
-    "fontawesomefree==6.1.1",
+"amqp==2.6.1"
+"asgiref==3.7.2"
+"attrs==23.1.0"
+"Automat==22.10.0"
+"backports.zoneinfo==0.2.1"
+"beautifulsoup4==4.12.2"
+"billiard==3.6.4.0"
+"black==22.3.0"
+"bleach==6.0.0"
+"bsddb3==6.2.7"
+"celery==4.4.7"
+"certifi==2023.5.7"
+"cffi==1.15.1"
+"chardet==5.1.0"
+"charset-normalizer==3.1.0"
+"click==8.1.3"
+"click-didyoumean==0.3.0"
+"click-repl==0.2.0"
+"constantly==15.1.0"
+"coreapi==2.3.3"
+"coreschema==0.0.4"
+"coverage==7.2.7"
+"cryptography==41.0.1"
+"dj-pagination==2.5.0"
+"Django==4.2.2"
+"django-appconf==1.0.5"
+"django-compressor==4.3.1"
+"django-debug-toolbar==4.1.0"
+"django-extensions==3.2.3"
+"django-formtools==2.4.1"
+"django-gravatar2==1.4.4"
+"django-js-asset==2.0.0"
+"django-libsass==0.9"
+"django-mptt==0.14.0"
+"django-nested-admin==4.0.2"
```

Comment:
I don't think this is right (pasted by mistake?):
- no commas
- there are two ``requirements``
- indirect dependencies are included

Reply:
Yep, confirmed the fragment was mistaken